docs/developers/built-in-functions.md

```diff
@@ -115,3 +118,74 @@ already covered [here](/developers/account-storage).
 
 Most of the ESDT and NFT related built-in function are already described in the [ESDT](/developers/esdt-tokens/) and
 [NFT](/developers/nft-tokens) sections.
+
+[comment]: # (mx-context-auto)
+
+## **SetGuardian**
+
+`SetGuardian` is used to set a guardian for a given address. 
+
+```rust
+SetGuardianTransaction {
+    Sender: <sender address>
+    Receiver: <sender address>
+    Value: 0
+    GasLimit: 1_000_000
+    Data: "SetGuardian" +
+          "@" + <guardian address in hexadecimal encoding> +
+          "@" + <service ID in hexadecimal encoding>
+}
+```
+
+_For more details about how arguments have to be encoded, check [here](/developers/sc-calls-format)._
+
+:::note
+1. After sending a `SetGuardian` transaction, the user has to wait for **20 epochs** until the chosen guardian becomes `activeGuardian` (`activationEpoch` field will show the epoch when a `pendingGuardian` will become active). Until the `activationEpoch` is reached, the chosen guardian will be seen as `pendingGuardian`. See [Guardian Data](/sdk-and-tools/rest-api/addresses) for more information.
+2. In the case an address already has an activeGuardian, and the user wants to change it, a `SetGuardian` transaction has to be sent again and **20 epochs** have to be awaited.
+3. If the account is already Guarded (see below `GuardAccount` transaction), a user can _Fast SetGuardian_ (not waiting for the **20 epochs** anymore) by sending a guarded transaction.
+:::
+
+[comment]: # (mx-context-auto)
+
+## **GuardAccount**
+
+`GuardAccount` is used to guard the account. In other words, after sending this transaction (and if it is successfull), all transactions that will be sent afterwards by this address must have `guardian` and `guardianSignature` filled in until a successfull `UnGuardAccount` transaction is being sent. 
+
+```rust
+GuardAccountTransaction {
+    Sender: <sender address>
+    Receiver: <sender address>
+    Value: 0
+    GasLimit: 1_000_000
+    Data: "GuardAccount"
+}
+```
+
+_For more details about how arguments have to be encoded, check [here](/developers/sc-calls-format)._
+
+:::note
+After sending a successfull `GuardAccount` transaction, only co-signed (guarded) transactions can be sent. Any other transaction will not be processed, except a new `SetGuardian` (for which **20 epochs** must pass until the changes will occur).
+:::
+
+[comment]: # (mx-context-auto)
+
+## **UnGuardAccount**
+
+`UnGuardAccount` is used to unguard the account. After a successfull transaction, it is not mandatory anymore for the user to send co-signed (guarded) transactions.
```

Comment:
maybe add a note, that UnGuardAccount can only be done through a guarded transaction, when you have access to your activated guardian, otherwise you need to change the guardian first.

Reply:
done.